app/src/main/java/mozilla/org/webmaker/view/WebmakerWebView.java

```diff
@@ -25,6 +25,6 @@ public WebmakerWebView(Context context, Activity activity, String pageName, JSON
         this.load("file:///android_asset/www/pages/" + pageName + "/index.html", null);
         this.setResourceClient(new WebClient(this));
         this.setBackgroundColor(getResources().getColor(R.color.light_gray));
-        this.addJavascriptInterface(new WebAppInterface(context, routeParams), "Android");
+        this.addJavascriptInterface(new WebAppInterface(context, activity, routeParams), "Android");
```

Comment:
You don't need to pass along the `Activity` here. You can cast the context instead:
```java
Activity activity = (Activity) context;
```